pyproject.toml

```diff
@@ -74,12 +74,20 @@ redshift = ["psycopg2"]
 snowflake = ["snowflake-connector-python", "cryptography"]
 presto = ["presto-python-client"]
 oracle = ["oracledb"]
-mssql = ["pyodbc"]
+mssql = ["pyodbc"]  # natively supported in Datafold Cloud only
 # databricks = ["databricks-sql-connector"]
 trino = ["trino"]
 clickhouse = ["clickhouse-driver"]
 vertica = ["vertica-python"]
 duckdb = ["duckdb"]
+all-local-dbs = [
+    "preql", "mysql-connector-python", "psycopg2", "snowflake-connector-python", "cryptography", "presto-python-client",
+    "oracledb", "trino", "clickhouse-driver", "vertica-python", "duckdb"
+]
+all-cloud-dbs = [
```

Comment:
Remove this as it's misleading. Not all of these are supported in Datafold Cloud